views/payment.haml

```diff
@@ -0,0 +1,12 @@
+- if @payment.successful?
+  .govuk-panel.govuk-panel--confirmation
+    %h2.govuk-panel__title
+      Payment complete
+    .govuk-panel__body
+      Your reference number
+      %br
+      %strong= @payment.govpay_reference
+  %p
```

Comment:
should the %p be moved out of if / else, so that the 'back link' appears regardless ? A tiny thing :)

Reply:
ok, that's cool